<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>maproc v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="MenuView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="Overview">
       <attribute name="title">
        <string>Overview</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="infosView">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
          <row>
           <property name="text">
            <string>Name</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Umask</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>State</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Tgid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Ngid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Pid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>PPid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>TracerPid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Uid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Gid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>FDSize</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Groups</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>NStgid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>NSpid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>NSpgid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>NSsid</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Infos_pid</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Mapping">
       <attribute name="title">
        <string>Mapping</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="grid_Mapping">
          <item row="4" column="3">
           <layout class="QFormLayout" name="formmap">
            <item row="1" column="0">
             <widget class="QLineEdit" name="find">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="searchButton">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>search</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QComboBox" name="type">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <item>
               <property name="text">
                <string>char</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>int</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>int8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>int16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>uint32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>uint64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>float</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>float32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>float64</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pidButton">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>40</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>PID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTableWidget" name="infos_addr">
            <property name="minimumSize">
             <size>
              <width>346</width>
              <height>90</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>346</width>
              <height>90</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::DashLine</enum>
            </property>
            <row>
             <property name="text">
              <string>Heap</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Stack</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Address_on</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address_off</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Size_mapped</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QTableWidget" name="viewAddress">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1300</width>
              <height>1300</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>500</width>
              <height>100</height>
             </size>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::DashLine</enum>
            </property>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>close</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cleanButton">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>clean</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
